qcodes/tests/__init__.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+import qcodes as qc
+
+
+@pytest.fixture(scope="session", autouse=True)
+def disable_telemetry():
```

Comment:
In antisipation that we are going to also want to disable other things (subscribers at least) should we call this method something like `setup_config_for_testing`?

Reply:
Or is it cleaner to use one fixture per disabled feature?

Reply:
In any case, this has now been moved into `conftest.py` (Thanks, @Dominik-Vogel!)